desktop/assets/texts/de-DE/settings.ftl

```diff
@@ -45,7 +46,28 @@ align-top-right = Oben-Rechts
 align-bottom-right = Unten-Rechts
 align-force = Erzwingen
 scale-mode = Skalierungsmodus
+scale-mode-noscale = Unskaliert (100%)
+scale-mode-noscale-tooltip =
+    Zeigt den Film in der Originalgröße an, ohne Zoom.
```

Comment:
There's a period at the end of this sentence, but not at the end of the ones for the other scale modes.

Reply:
Fixed